### PR TITLE
fixed channel page title

### PR DESCRIPTION
**Pull request by susanm74**
BEFORE
Screen Shot 2022-08-05 at 2 51 40 PM
AFTER
Screen Shot 2022-08-10 at 2 05 45 PM

### CODE REVIEW

**Comment by ericnewcomer** [comment]
Looks like your icons are missing in your second screenshot... we can work that out over chat if still an issue.

One thing we want to be careful about throughout the UI is consistency. For creating new things, we have "New Contact", "New Group", "New Flow", etc. I know this was already set to "Add Channel", but these are the sorts of things to keep your eye out for across the product, so I'd change that for consistency's sake. Someday (soon?) we'll have some interface guidelines.

**Comment by ericnewcomer** [comment]
This PR is failing due to formatting issues. If you run `./code_check.py` in your PR branch, it'll run the same checks CI does so you can avoid these kinds of failures. Alternatively, you can set your environment up to auto-format.

**Review by norkans7 — APPROVED** [review]

File: temba/channels/views.py [hunk 1]
```diff
@@ -1243,6 +1243,8 @@ def get_context_data(self, **kwargs):
             context["only_regional_channels"] = only_regional_channels
             return context
 
+        title = _("Add Channel")
```
Comment (rowanseymour) [inline temba/channels/views.py:1246]: move declaration to above methods

Comment (ericnewcomer) [reply temba/channels/views.py:1246]: @susanm74

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**